src/index.js

```diff
@@ -705,6 +705,20 @@ class Josh {
     }
     return returnvalue;
   }
+
+  /**
+   * Allows you to iterate this db with for await...of loop.
```

Comment:
```suggestion
   * Async iterator.
   * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/asyncIterator}
```
We don't use "you" or any personal pronoun in documentation.